src/known-organizations/index.js

```diff
@@ -21,10 +22,16 @@ export const KnownOrganizations = {
       '0x635193983512c621e6a3e15ee1dbf36f0c0db8e0',
       { name: 'Aragon One', image: aragonOneImage },
     ],
+
+    // brightid.aragonid.eth
+    [
+      '0x5aad137d8f7d2dc6e1b2548c059b1483360bcc6a',
+      { name: 'BrightId', image: brightIdImage },
```

Comment:
I think the correct written capitalization is “BrightID” (and “brightID” in the logo):

```suggestion
      { name: 'BrightID', image: brightIdImage },
```

Reply:
Fixed in https://github.com/aragon/aragon/pull/894